log4j-core-test/src/test/java/org/apache/logging/log4j/core/appender/rolling/RollingAppenderUncompressedTest.java

```diff
@@ -41,16 +39,20 @@ public class RollingAppenderUncompressedTest {
 
     private final Logger logger = LogManager.getLogger(RollingAppenderUncompressedTest.class.getName());
 
-    @ClassRule
-    public static CleanFolders rule = new CleanFolders(CONFIG);
+    // @RegisterExtension
+    // private CleanFoldersRuleExtension cleanFolders = new CleanFoldersRuleExtension(
+    //         DIR,
+    //         CONFIG,
+    //         RollingAppenderUncompressedTest.class.getName(),
+    //         this.getClass().getClassLoader());
```

Comment:
We left this block commented out because it makes the test fail on GHA Windows:

``` 
Run 1: RollingAppenderUncompressedTest.testAppender target\rolling4 failed with java.nio.file.FileSystemException: target\rolling4\rollingtest.log: The process cannot access the file because it is being used by another process.
``` 

We found [the commit that introduced the `CleanFolders` changes](https://github.com/apache/logging-log4j2/commit/b1b734e2cc640b5788c6b656176f854df5584453#diff-5ddf1f34f0ea4dfb82f7e574b53a675de1f9e7013f7f1aeb90d10e01188fa1e5), it's passing the `CONFIG` but we think it should pass `DIR`. However, when refactored, it generates the described error so we would like to confirm that this is a bug and what would be the next steps. Thank you